app/elements/socobo-recipe/socobo-recipe-details-add.html

```diff
@@ -203,101 +184,99 @@
                   exit-animation="fade-out-animation"
                   with-backdrop>
       <!-- DIALOG -->
-      <!-- EXIT DIALOG-->
-      <paper-icon-button id="exit" class="style-exit-add-button" icon="clear" on-tap="_clearInputFields">
-      </paper-icon-button>
       <!-- RECIPE NAME -->
-      <paper-input
-        id="title_input"
-        class="inputs style-recipe-name"
-        label="Please name your recipe ... "
-        value="{{recipe.desc}}"
-        required
-        pattern=".+"
-        auto-validate
-        error-message="Missing name or invalid characters!"
-        autofocus="true"
-        focused="true">
-      </paper-input>
-
-      <!-- CONTENT -->
-      <div class="content">
-
-        <!-- LEFT COLUMN -->
-        <div class="leftColumn">
-          <!-- IMAGE SECTION -->
-          <input type="file" id="fileChooser" class="style-file-chooser"/>
-          <paper-material elevation="1" class="recipeImg" id="imageHolder">
-            <div id="overlay" class="overlay" on-tap="_openFileChooser">
-              <iron-icon icon="create"></iron-icon>
-            </div>
-            <iron-image
-              id="add_image"
-              class="recipeImg_image"
-              src="{{recipe.image}}"
-              sizing="cover"
-              preload="true"
-              fade
-              placeholder="images/placeholder/placeholder.png"><!-- Does not work with documentation.html -->
-            </iron-image>
-          </paper-material>
-
-          <!-- DETAIL INGREDIENTS -->
-          <div class="subHeadings">Ingredients:</div>
-          <hr class="separator"/>
-          <template is="dom-repeat" items="{{ingredient_inputs}}">
-            <socobo-recipe-details-ingredient
-              state="add"
-              ingredient="{{item}}"
-              measurements="[[measurements]]"
-              id="{{item}}"
-              on-rem_ingr="_removeIngr">
-            </socobo-recipe-details-ingredient>
-          </template>
-          <paper-fab
-            icon="add"
-            id="addIngr"
-            mini="true"
-            class="style-add-ingr-button"
-            on-tap="_addIngr">
-          </paper-fab>
-        </div>
-
-        <!-- RIGHT COLUMN -->
-        <div class="rightColumn">
-          <!-- DESCRIPTION SECTION -->
-          <div class="subHeadings">Description</div>
-          <hr class="separator"/>
-          <paper-textarea
-            id="text_input"
-            class="inputs"
-            rows="6"
-            label="Here you might place a brief description or the like ..."
-            value="{{recipe.text}}">
-          </paper-textarea>
-          <!-- DETAIL STEPS -->
-          <div class="subHeadings">Steps</div>
-          <hr class="separator"/>
-          <div class="steps" id="steps_input">
-            <template is="dom-repeat" items="{{recipe_inputs}}">
-              <socobo-recipe-details-step
+      <paper-dialog-scrollable class="scrollableContent">
```

Comment:
I moved the content which should be scrollable into the paper-dialog-scrollabe after I had built it in. The purpose of this was to create a button area which is stuck to the bottom and doesn't overlay other gui elements